agents/testutil/deployers.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/synapsecns/sanguine/ethergo/deployer"
 )
 
+// SynChainID the id of the SynChain.
+const SynChainID uint = 10
```

Comment:
fyi, not sure if we want to change this here or someone else but we should at least note as of #1308, this no longer needs to be hardcoded (a change we made for both #1274 and #1312)

Reply:
Added a TODO